Patch by Sergei Golovan adds call to odbc:start() to make ODBC working
in erlang R12B since ODBC application doesn't start automatically anymore.

Index: ejabberd-1.1.4/src/odbc/ejabberd_odbc.erl
===================================================================
--- ejabberd-1.1.4.orig/src/odbc/ejabberd_odbc.erl
+++ ejabberd-1.1.4/src/odbc/ejabberd_odbc.erl
@@ -212,6 +212,7 @@
 %% part of init/1
 %% Open an ODBC database connection
 odbc_connect(SQLServer) ->
+    odbc:start(),
     case odbc:connect(SQLServer,[{scrollable_cursors, off}]) of
 	{ok, Ref} -> 
 	    erlang:monitor(process, Ref),
